src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -2092,12 +2092,11 @@ frmelmnts.lbl.deleteSubmission = Are you sure you want to delete instance?
 frmelmnts.lbl.instanceName = Instance Name
 frmelmnts.lbl.notstarted = Not started
 # alertpop
-frmelmnts.alert.updateprofiletitle = Update Profile
 frmelmnts.alert.updateprofilecontent = Please update your profile
 frmelmnts.alert.uploadevidencetitle = Upload Evidence
 frmelmnts.alert.uploadevidencecontent = I have made sure that I do not violate others’ copyright, privacy or third party rights.
 frmelmnts.alert.evidence_content_policy = I confirm that the Content submitted by me comply with the terms and policies of DIKSHA, including 
-frmelmnts.alert.evidence_content_policy_label= DIKSHA's Content Policy
+frmelmnts.alert.evidenceContentPolicyLabel= DIKSHA's Content Policy
```

Comment:
do not use Diksha in sunbird instead use {instance} and replace instance in the code with the instance label